src/main/java/net/raphimc/viabedrock/protocol/rewriter/ItemRewriter.java

```diff
@@ -120,9 +121,14 @@ public Item javaItem(final BedrockItem bedrockItem) {
                     rewriter = metaItemRewriter.get(meta);
                 }
             } else {
-                ViaBedrock.getPlatform().getLogger().log(Level.WARNING, "Missing bedrock -> java item mapping for " + identifier);
```

Comment:
Put the warning in an else block (It should still warn if the user didn't load the resource pack)